zipkin/src/main/java/com/linecorp/armeria/client/tracing/HttpTracingClient.java

```diff
@@ -68,7 +73,13 @@
     public static Function<Client<HttpRequest, HttpResponse>, HttpTracingClient> newDecorator(
             Tracing tracing,
             @Nullable String remoteServiceName) {
-        ensureScopeUsesRequestContext(tracing);
+        try {
+            ensureScopeUsesRequestContext(tracing);
+        } catch (IllegalStateException e) {
+            logger.warn("{} - it is appropriate to ignore this warning if this client is not being used " +
```

Comment:
nit: How about adding a test case for this? e.g, We may build a `Tracing` with `ThreadLocalCurrentTraceContext`, then confirm that `ensureScopeUsesRequestContext` raises an exception and `HttpTracingClient.newDecorator()` doesn't.

Reply:
Added back the tests I removed - they seemed a bit weird since they don't do anything but guess it doesn't hurt.